README.md

```diff
@@ -1 +1,49 @@
-# node-hello-world-lab
\ No newline at end of file
+# Node Hello World
+
+## Objectives
+
+1. Implement Hello World in Node
+1. Run the script from the terminal
+
+
+## Introduction
+
+In this lab, you'll practice how to run Node scripts from command line. In addition, you'll develop your first Node application—Hello World, the traditional first app in any language.
+
+## Instructions
+
+1. Create file `hello-world.js` and open it in your favorite editor
+2. Implement an expression (function with `return`) which returns "Hello World"
+3. Export your expression (function with `return`) with `module.exports = ...`
```

Comment:
give context here: "We need to export our function so that our specs have access to this function to test the return value". You do that by doing `module.exports = helloWorld` "(they already know the name of the function they're defining so we can just give them this line